manifest.php

```diff
@@ -48,7 +48,7 @@
     'label' => 'Proctoring',
     'description' => 'Proctoring for deliveries',
     'license' => 'GPL-2.0',
-    'version' => '19.2.1',
+    'version' => '19.2.2',
     'author' => 'Open Assessment Technologies SA',
     'requires' => array(
         'tao'            => '>=38.9.0',
```

Comment:
You should update this line as well, since the fix is relying on a packaged installed by taoCore in `41.7.1`